cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/ProvisioningService.java

```diff
@@ -692,7 +692,9 @@ private void runWithProgramLogging(ProgramRunId programRunId, Map<String, String
       return Optional.empty();
     }
 
+    LOG.debug("Cancelling {} task for program run {}.", taskKey.getType(), taskKey.getProgramRunId());
```

Comment:
should these be trace?

Reply:
hm, this one and the one about storing state should probably be trace, but I think the message about a task being cancelled should be debug, as it's similar to the other messages we have about executing different provision subtasks